packages/core/src/sass/currency-select-radio.scss

```diff
@@ -77,6 +77,15 @@
     &__radio-button {
         display: none;
     }
+    &__popover {
+        position: relative;
+        left: 4.8rem;
+        top: -4.8rem;
+        margin-top: -16px;
+    }
+    &__popover-container {
+        z-index: 9999 !important;
```

Comment:
I see that there is a zIndex prop that can be passed to the `< Popover />`, could that be used instead? then we don't need to use !important